content/docs/concepts/security/index.md

```diff
@@ -10,197 +10,230 @@ aliases:
-  communication
+{{< image width="80%" ratio="56.25%"
+    link="./overview.svg"
+    alt="Istio security overview."
+    caption="Istio Security Architecture"
+    >}}
+
+Istio security provides strong identity, powerful policy, transparent TLS encryption and AAA (authentication/authorization/audit)
+to protect your services and data. The goals of Istio security are:
+
+- **Security by default**: no changes needed for application code and infrastructure
+
+- **Defense in depth**: integrate with existing security systems to provide multiple layers of defense
 
-- Providing a key management system to automate key and certificate generation,
-  distribution, rotation, and revocation
+- **Zero-trust network**: build security solutions on untrusted networks
 
-The following diagram shows Istio's security architecture, which includes three
-primary components: identity, key management, and communication security. The
-diagram shows how to use Istio to secure the service-to-service communication
-between the `frontend` service running as the `frontend-team` service account
-and the `backend` service running as the `backend-team` service account. Istio
-supports services running on Kubernetes containers, virtual machines, and
-bare-metal machines.
+This guide provides a high-level overview of Istio security; you can find more information about specific Istio security concepts
+in other guides in this section.
+To find out how to get started using Istio security with deployed services, see the [Mutual TLS Migration](/docs/tasks/security/mtls-migration/).
+For more detailed how-to on using security features, see our [Security Tasks](/docs/tasks/security/).
 
-{{< image width="60%" ratio="52.44%"
-    link="./auth.svg"
-    alt="Components making up the Istio security model."
+## High-level architecture and features
+
+Istio security involves multiple components, including Citadel for key and certificate management,
+Sidecar and perimeter proxies for implementing secure communication between clients and servers,
+Pilot for distributing [authentication policies](/docs/concepts/security/#authentication-policies)
+and [secure naming information](/docs/concepts/security/#secure-naming) to the proxies,
+and Mixer for managing authorization and auditing.
+
+{{< image width="80%" ratio="56.25%"
+    link="./architecture.svg"
+    alt="Istio architecture."
     caption="Istio Security Architecture"
     >}}
 
-As illustrated in the diagram, Istio uses secret volume mounts to deliver
-keys and certificates from Citadel to Kubernetes containers. For services
-running on VMs or bare-metal machines, we introduce a node agent, which is a
-process running on each VM or bare-metal machine. The node agent generates the
-private key and the CSR (Certificate Signing Request) locally, sends the CSR to
-Citadel for signing, and delivers the generated certificate together with the
-private key to Envoy.
+### Identity provision
+
+Istio PKI, built on top of Istio Citadel,
```

Comment:
To add clarity I suggest:
The Istio PKI is built on top of Istio Citadel and securely provisions strong workload identities to to every workload.